engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/task/AbstractTaskProcessor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.processing.bpmn.task;
+
+import io.camunda.zeebe.engine.processing.bpmn.BpmnElementContext;
+import io.camunda.zeebe.engine.processing.bpmn.BpmnElementProcessor;
+import io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnBehaviors;
+import io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnIncidentBehavior;
+import io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnStateTransitionBehavior;
+import io.camunda.zeebe.engine.processing.deployment.model.element.ExecutableActivity;
+
+public class AbstractTaskProcessor implements BpmnElementProcessor<ExecutableActivity> {
```

Comment:
❌ We had some internal discussion about the naming `Abstract Task`. In other parts of Camunda we are using the term `Undefined Task`. In order to be consistent we should rename this to `UndefinedTaskProcessor`.